integrations/tensorflow/iree_tf_compiler/iree-import-tflite-main.cpp

```diff
@@ -37,6 +44,15 @@ int main(int argc, char **argv) {
                                              cl::value_desc("filename"),
                                              cl::init("-"));
 
+  // The output format flag is the master control for what we do with the
+  // in-memory compiled form.
+  llvm::cl::opt<OutputFormat> outputFormat(
+      "output-format", llvm::cl::desc("Format of imported output"),
+      llvm::cl::values(clEnumValN(OutputFormat::mlir_bytecode, "mlir-bytecode",
+                                  "MLIR Bytecode (default)"),
```

Comment:
We could have kept `mlir-ir` as the default, then gradually migrated to `mlir-bytecode`. A few things have broken because of this (https://github.com/iree-org/iree/issues/10405, https://buildkite.com/iree/iree-samples/builds/475). The bytecode format is new, so we should have expected a bit of instability. Unless we get even more reports of issues, I'm fine with not rolling back, but I think we should wait a bit before including this change in a stable release.